README.rst

```diff
@@ -92,12 +125,37 @@ Testing locally
 Deploying
 =========
 
-TODO: How can a new user go about deploying this component? Is it just a few
-commands? Is there a larger how-to that should be linked here?
+After setting up required configuration, events are produced using the
+``openedx_events.get_producer().send()`` method which needs to be called from
+the producing side. For more information, visit this `link`_.
+
+.. _link: https://openedx.atlassian.net/wiki/spaces/AC/pages/3508699151/How+to+start+using+the+Event+Bus#Producing-a-signal
+
+To consume events, openedx_events provides a management command called
+``consume_events`` which can be called like so:
+
+.. code-block:: bash
+
+   # consume XBLOCK_DELETED signal
+   python manage.py consume_events --signal org.openedx.content_authoring.xblock.deleted.v1 --topic xblock-deleted --group_id test_group --extra '{"consumer_name": "test_group.c1"}'
+
+   # replay events from specific redis msg id
+   python manage.py consume_events --signal org.openedx.content_authoring.xblock.deleted.v1 --topic xblock-deleted --group_id test_group --extra '{"consumer_name": "test_group.c1", "last_read_msg_id": "1679676448892-0"}'
```

Comment:
We might need to explain why someone may want to do that, and how to get that message id to know where to start from, or point to docs about it